cmd/kwil-admin/nodecfg/toml.go

```diff
@@ -103,6 +103,18 @@ format = "{{ .Logging.Format }}"
 # Time format: "epochfloat" (default), "epochmilli", or "rfc3339milli"
 time_format = "{{ .Logging.TimeEncoding }}"
 
+#######################################################################
+###                    Migration Config Options                     ###
+#######################################################################
+
+[migration]
```

Comment:
Do we need a whole migration section here ? or just migrate_from as the top level config? any opinions?

Reply:
This seems fine to me